packages/graphql/fragments/groupFieldsFragment.js

```diff
@@ -91,7 +91,7 @@ export default gql`
       neighborhood
       region
     }
-    members(first: 8, sortBy: "name", order: "desc") {
+    members(first: 4, sortBy: "last_active_at", order: "asc") {
```

Comment:
this should be desc, so its showing most recently active. about to do this on web too

Reply:
will update, or you could update here then just use this fragment ;)